pmlc/conversion/comp_to_llvm/passes.td

```diff
@@ -17,4 +17,13 @@ def ConvertCompToLLVM : Pass<"pmlc-convert-comp-to-llvm", "mlir::ModuleOp"> {
   ];
 }
 
+def ConvertCompToLevelZero : Pass<"pmlc-convert-comp-to-level-zero", "mlir::ModuleOp"> {
```

Comment:
This should be deleted.

Reply:
Removed as we use comp_to_llvm now, but i wonder if we shall keep a separate pass for level zero as we may need some info such as where to add submit and when we can release an event.

Reply:
This is a good point. I think we should consider adding a new pass to add submit Ops in the future (and also for vulkan).

Reply:
We definitly will have L0 specific passes including the one discussed here to improve performance.

Reply:
If they use comp_to_llvm with prefix for all targets, we may need to add a pass before or after that to add more target specific changes